test/livebook/storage/ets_test.exs

```diff
@@ -3,6 +3,16 @@ defmodule Livebook.Storage.EtsTest do
 
   alias Livebook.Storage.Ets
 
+  setup_all do
+    # clear all possible stale test's entities
+    [:insert, :fetch_key, :fetch, :delete, :all, :persistence]
+    |> Enum.each(fn namespace ->
+      namespace
+      |> Ets.all()
+      |> Enum.each(& Ets.delete(namespace, &1.id))
+    end)
```

Comment:
I'm wondering if we should remove the storage file entirely before starting the pap in `test_helper.exs`. At the same time, @josevalim?

Reply:
We will have to delete it in config/test.exs. The app is already started on test_helper.